integration-tests/mongodb-panache/src/test/java/io/quarkus/it/mongodb/panache/profiles/MongoDatabaseResolverTestProfile.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.it.mongodb.panache.profiles;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+public class MongoDatabaseResolverTestProfile implements QuarkusTestProfile {
```

Comment:
Does this serve an actual purpose since it doesn't override anything from the parent type?

Reply:
It's a mistake, I was not using it anymore. The initial idea was mapping some properties to be used for the test using this profiles. I'll remove it